flink-runtime/src/main/java/org/apache/flink/runtime/state/StateBackend.java

```diff
@@ -220,7 +222,8 @@
 			kvStateRegistry,
 			ttlTimeProvider,
 			new UnregisteredMetricsGroup(),
-			stateHandles);
+			stateHandles,
+			new CloseableRegistry());
```

Comment:
Same here, I think this method only is reached by tests and has no justification to exist in the interface. If production code would accidentally call it, certain things like cancelling a restore will not work properly. I think it is better to remove.

Reply:
Ditto.